apirunner/apirunner.go

```diff
@@ -124,11 +125,12 @@ func executeAPIandCalculate(profileName string, apiURL string, command string, p
 	var avgTime float64
 	var totalTime float64
 	var count float64
+	isGetRequest, _ := regexp.MatchString("^(get|list|query|find)(\\w+)+$", command)
```

Comment:
```suggestion
	isGetRequest, _ := regexp.MatchString("^(get|list|query|find|ready)(\\w+)+$", command)
```

for readyForShutdown call

Reply:
also, this quota calls - _quotaIsEnabled, quotaTariffList, quotaSummary, quotaBalance, quotaStatement_ can be added to GET list.

Reply:
@kevin-lii can you check and address this ^^^